dotnet/src/Connectors/Connectors.OpenAI/OpenAIPromptExecutionSettings.cs

```diff
@@ -156,7 +154,6 @@ public int ResultsPerPrompt
     /// <remarks>
     /// Possible values are: "json_object", "text", <see cref="ChatCompletionsResponseFormat"/> object.
     /// </remarks>
-    [Experimental("SKEXP0010")]
```

Comment:
Are we still ok with this being typed as "object"?

Reply:
Will investigate some more before graduating this one